cmd/kube-scheduler/app/server.go

```diff
@@ -155,86 +145,86 @@ func runCommand(cmd *cobra.Command, args []string, opts *options.Options) error
 
 	// Configz registration.
 	if cz, err := configz.New("componentconfig"); err == nil {
-		cz.Set(cc.ComponentConfig)
+		cz.Set(opts)
```

Comment:
I think this should be `cz.Set(opts.ComponentConfig)` to keep the current functionality.

Reply:
Done